shell/platform/android/io/flutter/app/FlutterPluginRegistry.java

```diff
@@ -69,16 +71,16 @@ public Registrar registrarFor(String pluginKey) {
         return new FlutterRegistrar(pluginKey);
     }
 
-    public void attach(FlutterView flutterView, Activity activity) {
-        mFlutterView = flutterView;
+    public void attach(ViewRegistry viewRegistry, Activity activity) {
```

Comment:
I don't think we can change a public API

Reply:
This change should be non breaking since FlutterView implements ViewRegistry (so if anyone is extending it, it should not cause any build failure).